scripts/install_third_party_libs.py

```diff
@@ -210,6 +209,19 @@ def ensure_pip_library_is_installed(package, version, path):
         pip_install(package, version, exact_lib_path)
 
 
+def get_yarn_command():
+    """Get command for yarn."""
+    # Rename the one without extension, otherwise it will be executed.
```

Comment:
Slightly confused about what's going on here...

- Typically, "get" operations should not modify stuff. This seems to have a major side effect.
- Why will `yarn` be executed if it's not renamed?